libs/NfcCoreLib/lib/LibNfc/phLibNfc_Iso15693.c

```diff
@@ -13,21 +13,115 @@
 #define PHLIBNFC_GETSYSINFO_RESP_MINLEN         (0x02)
 
 
-/* This function shall send Get system information command to ISO15693 tag */
-static NFCSTATUS phLibNfc_GetSysInfoCmd(void *pContext, NFCSTATUS status, void *pInfo);
+static NFCSTATUS phLibNfc_InventoryCmd(void *pContext, NFCSTATUS status, void *pInfo);
+static NFCSTATUS phLibNfc_InventoryResp(void *pcontext, NFCSTATUS status, void *pInfo);
 
-/* This function shall Process system information received from ISO15693 tag */
+static NFCSTATUS phLibNfc_GetSysInfoCmd(void *pContext, NFCSTATUS status, void *pInfo);
 static NFCSTATUS phLibNfc_GetSysInfoResp(void *pcontext, NFCSTATUS status, void *pInfo);
 
-static NFCSTATUS phLibNfc_GetSysInfoComplete(void *pContext,NFCSTATUS status,void *pInfo);
+static NFCSTATUS phLibNfc_GetTagInfoComplete(void *pContext,NFCSTATUS status,void *pInfo);
 
-phLibNfc_Sequence_t gphLibNfc_Iso15693GetSysInfoSeq[] = {
+phLibNfc_Sequence_t gphLibNfc_Iso15693GetTagInfoSeq[] = {
+    {&phLibNfc_InventoryCmd, &phLibNfc_InventoryResp},
     {&phLibNfc_GetSysInfoCmd, &phLibNfc_GetSysInfoResp},
-    {NULL, &phLibNfc_GetSysInfoComplete}
+    {NULL, &phLibNfc_GetTagInfoComplete}
 };
 
 static void phLibNfc_Iso15693Sequence(void* pContext,NFCSTATUS status,pphNciNfc_Data_t pInfo);
 
+static NFCSTATUS phLibNfc_InventoryCmd(void *pContext,NFCSTATUS status,void *pInfo)
+{
+    NFCSTATUS wStatus = status;
+    pphLibNfc_Context_t pCtx = (pphLibNfc_Context_t ) pContext;
+    phNciNfc_TransceiveInfo_t tIso15693Info;
+    pphNciNfc_DeviceInfo_t pDevInfo = NULL;
+    uint8_t aGetSysInfoCmdBuff[5] = {0};
+    uint32_t dwIndex = 0;
+    PH_LOG_LIBNFC_FUNC_ENTRY();
+    UNUSED(pInfo);
+    if(NULL != pCtx)
+    {
+        pDevInfo = (pphNciNfc_DeviceInfo_t)pCtx->pInfo;
+        if((NULL != pDevInfo->pRemDevList[dwIndex]) &&
+            (phNciNfc_eISO15693_PICC == pDevInfo->pRemDevList[dwIndex]->RemDevType))
+        {
+
+            // Request flags
+            aGetSysInfoCmdBuff[0] = 0x26;
```

Comment:
What is the meaning of 0x26 here ?

Reply:
Request flags for Inventory ISO 15693 command:

> Bit 3: Inventory_flag
> Bit 6: Nb_slots_flags
> Bit 7: Option_flag
>